src/utils/log-util.ts

```diff
@@ -0,0 +1,22 @@
+import { LogLevel } from '../enums';
+
+/**
+ * Logs information about different actions taking place to console.
+ *
+ * @param functionName - the name of the function where the logging is happening
+ * @param message - the message to write to console
+ * @param logLevel - the severity level of the log
+ */
+export const log = (functionName: string, message: string, logLevel: LogLevel = LogLevel.LOG) => {
```

Comment:
Can we match the spread nature of `console.log` here to make it easier to use in the future.

```suggestion
export const log = (functionName: string, logLevel: LogLevel, ...message: any[]) => {
```